backend/de.metas.swat/de.metas.swat.base/src/test/java/de/metas/invoicecandidate/document/dimension/InvoiceCandidateDimensionFactoryTest.java

```diff
@@ -16,6 +20,9 @@
 	void beforeEach()
 	{
 		AdempiereTestHelper.get().init();
+
+		final List<DimensionFactory<?>> dimensionFactories = new ArrayList<>();
+		dimensionFactories.add(new InvoiceCandidateDimensionFactory());
```

Comment:
what's the purpose of this code chunk? (2)

Reply:
Leftover from debugging. Deleted (2)